resources/appmesh-mesh.go

```diff
@@ -0,0 +1,64 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/appmesh"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+)
+
+type AppMeshMesh struct {
```

Comment:
I am not sure about the naming here, I think `AppMesh` should be enough.

Reply:
I opted for this given that AppMesh is the service name and Mesh is an API resource within the service. Similar to `APIGatewayV2API` ([ref](https://github.com/rebuy-de/aws-nuke/blob/main/resources/apigatewayv2-apis.go#L10)) is not `APIGatewayV2` named only.

I don't have a strong opinion, and are happy to change if you feel `AppMesh` is the appropriate name.

Reply:
Valid point, let's keep it like this.